framework/sandstone.cpp

```diff
@@ -1831,9 +1831,21 @@ static int spawn_child(const struct test *test, intptr_t *hpid, int shmempipefd)
     snprintf(statefdstr.begin(), statefdstr.size(), "%u", unsigned(statefd));
 
     std::string random_seed = random_format_seed();
+#ifdef _WIN32
+    // _spawn on Windows requires argv elements to be quoted if they contain space.
+    std::string argv0_str;
+    argv0_str.reserve(strlen(program_invocation_name) + 2);
+    argv0_str += '\"';
+    argv0_str += program_invocation_name;
+    argv0_str += '\"';
+
+    const char * const argv0 = argv0_str.c_str();
```

Comment:
Alternatively, since we know that the child process will make no decisions based on the path, we could pass the portion after the last slash, or even a fixed string. For example, just set
```c++
    static const char argv0[] = SANDSTONE_EXECUTABLE_NAME ".exe";
```

Reply:
> Alternatively, since we know that the child process will make no decisions based on the path

I had similar thing in mind for a second, but thought brining this "dependency" to the code might not be right.
It's just hard to express in code that we are doing it and why. On the other hand I can't think of anything that will require opendcdiag to rely on the directory it was run from or argv[0] being equal to argv[0] of It's parent.
I will just leave a comment why we are doing it and stick with your suggestion.